tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -645,14 +646,14 @@ def fit(self,
         ag_args_ensemble = kwargs['ag_args_ensemble']
         excluded_model_types = kwargs['excluded_model_types']
 
-        quantile_levels = kwargs.get('quantile_levels', None)
-        if isinstance(quantile_levels, float):
-            quantile_levels = [quantile_levels]
-        if isinstance(quantile_levels, Iterable):
-            quantile_levels = np.sort(np.array(quantile_levels))
-        self._learner.quantile_levels = quantile_levels
-        if quantile_levels is not None and self._learner.problem_type is None:
-            self._learner.problem_type = QUANTILE
+        # quantile_levels = kwargs.get('quantile_levels', None)
```

Comment:
Why commented out code? Can this be removed?

Reply:
oh, sorry. this will be removed